src/components/Navigation/Navigation.css

```diff
@@ -0,0 +1,26 @@
+.navigation {
+  display: flex;
+  gap: 10px;
+  border: 3px solid #1b60aa;
+  border-radius: 10px;
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.nav-links {
```

Comment:
If we introduce a new NavLink component(mention down in Navigation.tsx file👇) we can isolate their style as well.